src/conponents/Footer/Footer.tsx

```diff
@@ -0,0 +1,59 @@
+import { Status } from '../../types/Status';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  filterBy: Status;
+  setFilterBy: (filterBy: Status) => void;
+}
+
+export const Footer: React.FC<Props> = ({ todos, filterBy, setFilterBy }) => {
+  const activeTodos = todos.filter(todo => !todo.completed);
+  const completedTodos = todos.filter(todo => todo.completed);
```

Comment:
You can use one iteration for dividing todos array to complete and uncomplete ( for future)